specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/ServerConfigurationOptions.json

```diff
@@ -0,0 +1,264 @@
+            "required": true,
+            "type": "string",
+            "enum": [
+              "allowPolybaseExport"
+            ],
+            "x-ms-enum": {
+              "name": "ServerConfigurationOptionName",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the specified managed instance server configuration option.",
+            "schema": {
+              "$ref": "#/definitions/ServerConfigurationOption"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidServerConfigurationOptionName - User specified server configuration option name that is not supported.\n\n * 400 InvalidServerConfigurationOptionValue - Server configuration option value \"{0}\" is not supported for server configuration option \"{1}\".\n\n * 400 InvalidServerConfigurationOptionProperties - The provided set of parameters in the body of the request is either empty or invalid.\n\n * 404 SubscriptionNotFound - The requested subscription was not found.\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-examples": {
+          "Gets managed instance server configuration option.": {
+            "$ref": "./examples/ServerConfigurationOptionGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ServerConfigurationOptions"
+        ],
+        "description": "Updates managed instance server configuration option.",
+        "operationId": "ServerConfigurationOptions_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "name": "serverConfigurationOptionName",
+            "in": "path",
+            "description": "The name of the server configuration option.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "allowPolybaseExport"
+            ],
+            "x-ms-enum": {
+              "name": "ServerConfigurationOptionName",
+              "modelAsString": true
+            }
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "Server configuration option parameters.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServerConfigurationOption"
+            }
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
>v1 [](http://example.com/codeflow?start=37&length=2)

use v3 everwhere

Reply:
@jamestao @ericshape  Other APIs seem to use v1 as well, and when I ran GenerateSwagger.ps1 script it automatically put v1.  Should we use v3 for this API and if we should how we can automatically generate it with GenerateSwagger.ps1 script?

Reply:
@rkmanda for SQL swagger now, we still use V1. V3 is a huge bump up to the SQL RP. We will plan for bump up soon for the whole SQL RP's wagger. Can we have an exception for it in 2022-08-01-preview release? Thanks~